### ISSUE TITLE
Version 0.36.0 breaks DryRun pipeline action

### ISSUE THREAD

**Issue by atoa** [awslabs/aws-service-catalog-puppet#156]
After upgrading to version 0.36.0, the DryRun pipeline action breaks. Reverting to 0.35.0 fixes the issue with the same configuration. Here's a stack trace from CodeBuild:

```
INFO:root:looking at vpc-flow-logs-bucket 
INFO:/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/aws.py:Found product: {'Id': 'prodview-4dm35moh4lx4q', 'ProductId': 'prod-d7wuqo2o4y3nq', 'Name': 'vpc-flow-logs-bucket', 'Owner': 'central-it@customer.com', 'ShortDescription': 'Deploys a VPC Flows log bucket', 'Type': 'CLOUD_FORMATION_TEMPLATE', 'Distributor': 'central-it-team', 'HasDefaultPath': False, 'SupportEmail': 'central-it-team@customer.com', 'SupportDescription': 'Contact us on Chime for help \\#central-it-team', 'SupportUrl': 'https://wiki.customer.com/central-it-team/self-service/account-iam'} 
INFO:/usr/local/lib/python3.7/site-packages/betterboto/utils.py:list_provisioning_artifacts: {'ProductId': 'prod-d7wuqo2o4y3nq'} 
INFO:/usr/local/lib/python3.7/site-packages/betterboto/utils.py:searching, PageToken: FirstPage 
INFO:/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/aws.py:Getting path for product prod-d7wuqo2o4y3nq 
Traceback (most recent call last): 
  File "/usr/local/bin/servicecatalog-puppet", line 11, in <module> 
    sys.exit(cli()) 
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 764, in __call__ 
    return self.main(*args, **kwargs) 
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 717, in main 
    rv = self.invoke(ctx) 
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1137, in invoke 
    return _process_result(sub_ctx.command.invoke(sub_ctx)) 
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 956, in invoke 
    return ctx.invoke(self.callback, **ctx.params) 
  File "/usr/local/lib/python3.7/site-packages/click/core.py", line 555, in invoke 
    return callback(*args, **kwargs) 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli.py", line 34, in dry_run 
    cli_commands.dry_run(f) 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli_commands.py", line 64, in dry_run 
    all_launch_tasks = cli_command_helpers.deploy_launches(manifest, puppet_account_id) 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli_command_helpers.py", line 553, in deploy_launches 
    all_tasks = deploy_launches_task_builder(deployment_map, manifest, puppet_account_id, section) 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli_command_helpers.py", line 571, in deploy_launches_task_builder 
    region_name, 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli_command_helpers.py", line 610, in deploy_launches_task_builder_for_account_launch_region 
    launch_details.get('version'), 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/cli_command_helpers.py", line 591, in get_required_params 
    PathId=aws.get_path_for_product(service_catalog, product_id, portfolio), 
  File "/usr/local/lib/python3.7/site-packages/servicecatalog_puppet/aws.py", line 230, in get_path_for_product 
    response = service_catalog.list_launch_paths(ProductId=product_id) 
  File "/usr/local/lib/python3.7/site-packages/botocore/client.py", line 357, in _api_call 
    return self._make_api_call(operation_name, kwargs) 
  File "/usr/local/lib/python3.7/site-packages/botocore/client.py", line 661, in _make_api_call 
    raise error_class(parsed_response, operation_name) 
botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the ListLaunchPaths operation: No launch paths found for resource: prod-d7wuqo2o4y3nq 
```

**Comment by eamonnfaherty** [comment]
Thanks for reporting this issue.  I have removed the 0.36.0 release and am working on a fix.

**Comment by eamonnfaherty** [comment]
@atoa sorry for the delay with this fix.  

This should now be resolved in version 0.36.1
